tests/test_generic/tests/test_industry_mail_behavior.py

```diff
@@ -0,0 +1,33 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo.tests import get_db_name, tagged
+from .industry_case import IndustryCase
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged('post_install', '-at_install')
+class IndustryMailBehaviorTestCase(IndustryCase):
+    def test_user_notification_type(self):
+        for _ in self.installed_industries:
+            db_name = get_db_name()
+            if db_name.endswith('imported_no_demo'):
+                mails = self.env['mail.mail'].search([('model', 'not in', ['hr.expense', False])])
+                if mails:
+                    models = list(set(mails.mapped('model')))
+                    _logger.warning(
+                        "Mails are generated from models: %s — use correct context to prevent mail pollution.",
+                        ', '.join(models)
+                    )
+                return
+
+            # Ensure admin notification type remains correct
+            users = self.env['res.users'].search([('group_ids', 'in', [
+                self.env.ref('base.group_user').id,
+                self.env.ref('base.group_system').id,
+            ])])
+
+            if any(user.notification_type != "inbox" for user in users):
+                _logger.warning("Notification type should remain 'inbox' for all users.")
```

Comment:
Maybe more helpful to log the user id, something like
```suggestion
            for user in users:
                if user.notification_type != "inbox":
                    _logger.warning("Notification type should remain 'inbox' for all users, not the case for %s (%d).", user.name, user.id)
```